src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const messageElement = document.createElement('div');
+
+messageElement.innerHTML = `Promise was resolved!`;
+
+messageElement.classList.toggle('message');
+
+const errorMessageElement = document.createElement('div');
+
+errorMessageElement.innerHTML = `Promise was rejected!`;
+
+errorMessageElement.classList = 'message error-message';
+
+const firstPromise = () => {
+  const promise = new Promise((resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  });
+
+  return promise;
+};
+
+const secondPromise = () => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error());
+    }, 3000);
+  });
+};
+
+const promise1 = firstPromise();
+
+const promise2 = secondPromise();
+
+function success() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
const success = () => {
```

Reply:
@volodymyr-soltys97 Thank you for your comments, in the future I will stick to one way of creating functions. Have a nice evening!